datapreview.tex

```diff
@@ -126,16 +128,21 @@ \subsection{Data Preview 1}
 \end{figure}
 
 The  processing and preparation of ComCam data for DP1 will take place during the first half of 2025, with an expected DP1 release data in June or July 2025. 
-The planned data products for DP1 are presented in Table~\ref{tab:data-preview-summary}. 
+The planned data products for DP1 are presented in Table~\ref{tab:data-preview-summary}.
 DP1 is expected to include of the order of 2000 science images.
-As the processing progresses, it is conceivable that some individual exposures might not meet the quality control standard of being “science-grade, astronomically useful” images, and consequently, the exact number of images  and coadd depths in DP1 may change. 
+The processing of crowded fields in the centers of 47 Tuc and Fornax is on a best-effort basis for DP1.
+The center of 47 Tuc is saturated and so no coadd images will be produced for this region.
+There will be no cell-based coadds in DP1, nor \textit{ShearObject} catalog.
```

Comment:
This is definitive - DP1 ShearObject table not even a stretch goal? (Just checking.)

Reply:
Nope - no cell based coadds == no ShearObject catalog